centre-api/src/centre_api/services/access_requests.py

```diff
@@ -66,27 +68,46 @@ def process_access_request(cls, access_request_id, status):
             raise PermissionError(
                 f"User does not have permission to update access requests for app '{app_name}'."
             )
-
-        return cls._do_update_access_request(access_request, status)
+        with session_scope() as session:
+            access_request_response = None
+            app = access_request.app
+            auth_user_response = AuthApiService.get_user_by_id(access_request.user_auth_guid)
+            access_request_response = cls._do_update_access_request(access_request, status, session)
+            _queue_access_denied_email(session, app, auth_user_response, status)
```

Comment:
General name for the queue email here right?

we can call _queue_access_request_email

and in a function called _queue_access_request_email

we can have an if statement on the status
and if denied we can call _queue_access_denied_email

This way we can just extend the if statements and add a function for each email status.